routes/services.js

```diff
@@ -0,0 +1,291 @@
+import { PrismaClient } from '../generated/prisma/index.js';
+
+const prisma = new PrismaClient();
+
+class ArticleService {
+  async getAllArticles(query) {
+    const page = parseInt(query.page) || 1; // 페이지
+    const limit = parseInt(query.limit) || 10; // 노출 항목
+    const sort = query.sort || 'recent'; // 정렬 설정
+    const keyword = query.keyword || ''; // 키워드 설정
+    const skip = (page - 1) * limit; // 넘길 항목수
+
+    let orderBy;
+    switch (sort) {
+      case 'resect':
```

Comment:
```suggestion
      case 'recent':
```

오타가 있네요.

만약 요청 쿼리에서 저렇게 추출해서 관리하는 게 힘드시다면, 쿼리를 별도의 객체나 상수로 관리해보시는 것도 방법입니다. 아래는 간단한 예시에요.

```javascript
export const SORT_OPTIONS = {
  RECENT: 'recent',
  OLD: 'old'
}

switch (sort) {
  case SORT_OPTIONS.RECENT: ......
}